### ISSUE TITLE
Sanity checking

### ISSUE THREAD

**Issue by sunilk747** [librariesio/license-compatibility#45]
Hello all,
I did a little research and I tried to come up with a new license tree. Please refer the attachment. If anybody here could review it or add any comment's it would be really helpful. 
![screenshot from 2017-12-19 16-57-42](https://user-images.githubusercontent.com/7105965/34155199-660e788e-e4de-11e7-8da3-3693bdcde890.png)

**Comment by andrew** [comment]
Yeah looks good to me, very similar to https://www.dwheeler.com/essays/floss-license-slide.html and http://janelia-flyem.github.io/licenses.html

cc @tieguy for a second opinion
